### PR TITLE
Support for rails 6.0.x

### PR DESCRIPTION
**Pull request by aksh93**
- Adds gemfile for rails 6 and updates for Travis CI build
- Update migrations for differing Rails versions
- Removes code for unsupported Rails versions
- Bumps up rapidfire version for major release

### CODE REVIEW

File: app/services/rapidfire/base_service.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module Rapidfire
   class BaseService
-    if Rails::VERSION::MAJOR == 4 || Rails::VERSION::MAJOR == 5
+    if Rails::VERSION::MAJOR >= 4
       include ActiveModel::Model
     else
```
Comment (snkshukla) [inline app/services/rapidfire/base_service.rb]: @aksh93 If we are removing support for versions < 4, why not remove the else wherever we are checking versions >=4 ?

Comment (aksh93) [reply app/services/rapidfire/base_service.rb]: Yeah makes sense

File: app/controllers/rapidfire/attempts_controller.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module Rapidfire
   class AttemptsController < Rapidfire::ApplicationController
-    if Rails::VERSION::MAJOR == 5
+    if Rails::VERSION::MAJOR >= 5
       before_action :find_survey!
     else
       before_filter :find_survey!
```
Comment (snkshukla) [inline app/controllers/rapidfire/attempts_controller.rb]: @aksh93 We can remove before_filter completely with rails 4.2 and above. https://edgeguides.rubyonrails.org/4_2_release_notes.html#action-pack-notable-changes

Comment (aksh93) [reply app/controllers/rapidfire/attempts_controller.rb]: Alright, let's do that.

**Comment by aksh93** [comment]
@iffyuva shall we get this merged in?

**Pull request merged**